src/components/App/App.js

```diff
@@ -13,41 +14,68 @@ class App extends Component {
       movies: [],
       selectedMovie: {},
       isClicked: false,
-      error: ''
-    }
+      error: "",
```

Comment:
Do you need a `hasError` piece of state? Can you utilize the `error` piece of state that you have already to determine an other logic?